components/uniset/src/uniset.rs

```diff
@@ -67,9 +67,27 @@ impl UnicodeSet {
         }
     }
 
+    /// Returns a [Chunks](https://doc.rust-lang.org/std/slice/struct.Chunks.html) of each interval
+    /// in the inversion list.
+    ///
+    /// # Example:
+    ///
+    /// ```
+    /// use icu_unicodeset::UnicodeSet;
+    /// let example_list = vec![65, 68, 69, 70];
+    /// let example = UnicodeSet::from_inversion_list(example_list).unwrap();
+    /// let mut example_ranges = example.ranges();
+    /// assert_eq!(&[65, 68], example_ranges.next().unwrap());
+    /// assert_eq!(&[69, 70], example_ranges.next().unwrap());
+    /// assert_eq!(None, example_ranges.next());
+    /// ```
+    pub fn ranges(&self) -> Chunks<u32> {
```

Comment:
IMO, a function named `ranges` in UnicodeSet should return `Range<char>`.  I think you should probably make a function `as_inversion_list(&self) -> &[u32]`, and at the call site, if you need chunks, you can always do `set.as_inversion_list().chunks(2)`.  I'll let you decide whether to make the function `pub` or `pub(crate)`.

Reply:
Since `pub(crate)` was brought to my view from you, I decided to just remove the `ranges` function for now as we did in the L1 PR (I needed it as a way to expose the inversion list to the builder, but without exposing everything directly to the user)